EnglishTranslation-master/SKILL.tsv

```diff
@@ -6043,7 +6043,7 @@ SKILL_20151224_006042	Zucken: Enhance
 SKILL_20151224_006043	Increases the damage dealt on an enemy with [Zucken] by 1% per attribute level.
 SKILL_20151224_006044	Cyclone: Resistance
 SKILL_20151224_006045	Increases the chance to block harmful effects by 15% per attribute level during [Cyclone].
-SKILL_20151224_006046	Increases the STA reduction cycle by 0.05 seconds per attribute level while [Bunshin no Jutsu] is active.
+SKILL_20151224_006046	Increases the stamina reduction cycle of [Bunshin no Jutsu] by 0.05 seconds per attribute level.
```

Comment:
STA and Stamina are used interchangeably. 
Korean description does say that this change happens when the skill is active/is in effect.